src/routes/AddEditQuiz/index.tsx

```diff
@@ -69,139 +58,139 @@ export default function AddEditQuiz() {
         body: `There ${
           unSavedQuestionsCount > 1 ? `are ${unSavedQuestionsCount} empty questions` : 'is an empty question'
         }. 
-        Do you want to finish the quiz with empty questions or cancel and and complete those questions first?`,
-        okCallback: finishQuiz,
-        okText: 'Finish quiz',
-        cancelText: 'Cancel and complete questions',
+        Please complete those questions first.`,
+        okText: 'Cancel and complete questions',
+        cancelText: '',
       });
     } else {
-      finishQuiz();
+      await unDraftQuiz(quizInfo.quizId);
```

Comment:
Don't we plan to save data in the backend as well in this case?

Reply:
I'm saving the quiz as draft, As soon as user add the question it will be saved in db. Once user finish the quiz I'm undrafting the quiz